builder rename fails

The rename doubles up on the dir for the tmp file:

```
ring /tmp/ort.builder create replicas=3
ring /tmp/ort.builder add active=true capacity=1000 tier0=server1 tier1=z1 address0=127.0.0.1:8001 address1=127.0.0.2:8001 meta=onmetalv1
ID:        181f00653e5281fc
Active:    true
Capacity:  1000
Tiers:     server1
           z1
Addresses: 127.0.0.1:8001
           127.0.0.2:8001
Meta:      onmetalv1
/tmp/ort.builder.187494569
Rename /tmp/tmp/ort.builder.187494569 /tmp/ort.builder: no such file or directory.
make: *** [ring] Error 1
```

--- a/ring/main.go
+++ b/ring/main.go
@@ -673,5 +673,5 @@ func persist(r ring.Ring, b *ring.Builder, filename string) error {
 	if err = f.Close(); err != nil {
 		return err
 	}
-	return os.Rename(path.Join(dir, tmp), filename)
+	return os.Rename(tmp, filename)
 }

